data/test-cases/plugins/MS Teams sync/Slash commands/disconnect_bot_account.md

```diff
@@ -0,0 +1,42 @@
+---
+# (Required) Ensure all values are filled up
+name: "Disconnect-bot account slash command"
```

Comment:
Make the names consistent. Somewhere, you write account slash command, user slash command and somewhere you write only "command". Make this consistent.